### PR TITLE
Auth submit result code

### PR DESCRIPTION
**Pull request by Ation**
Merge after PR to PB

### CODE REVIEW

File: BlockSettleUILib/AuthAddressConfirmDialog.cpp [hunk 1]
```diff
@@ -102,12 +102,36 @@ void AuthAddressConfirmDialog::onError(const QString &errorText)
...
-void AuthAddressConfirmDialog::onAuthAddressSubmitError(const QString &address, const QString &error)
+void AuthAddressConfirmDialog::onAuthAddressSubmitError(const QString &address, const bs::error::AuthAddressSubmitResult statusCode)
 {
    progressTimer_.stop();
+
+   QString errorText;
+
+   switch (statusCode) {
+   case bs::error::AuthAddressSubmitResult::SubmitLimitExceeded:
+      errorText = tr("Your account has reach the limit of how many Authentication Addresses it may have in submitted state. Please verify your currently submitted addresses before submitting further addresses");
```
Comment (sergey-chernikov) [inline BlockSettleUILib/AuthAddressConfirmDialog.cpp:113]: "has reached", and please wrap around this line - it's too long.

File: BlockSettleUILib/AuthAddressConfirmDialog.cpp [hunk 2]
```diff
@@ -102,12 +102,36 @@ void AuthAddressConfirmDialog::onError(const QString &errorText)
...
+      break;
+   case bs::error::AuthAddressSubmitResult::ServerError:
+      errorText = tr("Server error. Please try again later.");
```
Comment (sergey-chernikov) [inline BlockSettleUILib/AuthAddressConfirmDialog.cpp:116]: Some texts are with periods, some are not. I think we can have all of them without periods for better readability. E.g. this one can be transformed to "Server error - please try again later".

File: BlockSettleUILib/BSTerminalMainWindow.cpp [hunk 3]
```diff
@@ -1368,6 +1368,7 @@ void BSTerminalMainWindow::openAuthManagerDialog()
 void BSTerminalMainWindow::openAuthDlgVerify(const QString &addrToVerify)
 {
    const auto showAuthDlg = [this, addrToVerify] {
+      authAddrDlg_->setModal(true);
```
Comment (sergey-chernikov) [inline BlockSettleUILib/BSTerminalMainWindow.cpp:1371]: Do we really need modality here? I mean, I guess no action in this dialog requires the rest of the interface to be blocked.

File: BlockSettleUILib/AuthAddressConfirmDialog.cpp (hunk 2 above)
Comment (Ation) [reply BlockSettleUILib/AuthAddressConfirmDialog.cpp:119]: Dots to end added where missing. Don't think that dash could improve readability of 6 words.

File: BlockSettleUILib/AuthAddressConfirmDialog.cpp (hunk 1 above)
Comment (Ation) [reply BlockSettleUILib/AuthAddressConfirmDialog.cpp]: done.

File: BlockSettleUILib/BSTerminalMainWindow.cpp (hunk 3 above)
Comment (Ation) [reply BlockSettleUILib/BSTerminalMainWindow.cpp:1371]: Also it is not required for a user to interact with any other dialog in the app if he went to manage auth addresses. Also it is really easy to lose this dialog in a background.

**Pull request merged**